build.sbt

```diff
@@ -8,14 +7,20 @@ import xerial.sbt.Sonatype._
 
 name := "play-html-compressor"
 
-version := "0.4-SNAPSHOT"
+version := "0.4.1"
```

Comment:
Version should be a snapshot version until Play 2.4 final will be released. So the version should be 0.4.1-SNAPSHOT